README.md

```diff
@@ -57,19 +57,47 @@ There are some required params you need:
 
 -   `SOME_SALT_FOR_ROUTER_PROXY` A single use salt for deploying the router proxy. Max 11 characters
 
-## Limitations
-
-This solution is a POC so it contains several limitation:
+## Supported Networks
 
-### Networks
+For running the task on a given network it is required the presence of 3 smart contracts, Hyperlane
+[`InterchainAccountRouter`](https://docs.hyperlane.xyz/docs/reference/applications/interchain-account),
+`MulticallFactory` and [`CreateX`](https://github.com/pcaversaccio/createx).
```

Comment:
I would say we need to add createx deploys in here as well

Reply:
Would probably add a script that can update the registry.json to include existing createx addresse from https://github.com/pcaversaccio/createx/blob/main/deployments/deployments.json

Reply:
The address is the same on every network, so I'll add a validation for checking the network is present on that json